tasks/03_create_key.yml

```diff
@@ -0,0 +1,40 @@
+---
+# In this Play an ssh key pair is created for login to the backup server and secure data transfer.
+# We do not want the possibly existing key to be used/distributed,
+# so a backup-specific (backup/backup.pub) one is created.
+- name: Create ssh-key (if neeeded) for {{ borgbackup_user }}
+  tags:
+    - install_backup
+  block:
+    - name: Ensure directory exist
+      ansible.builtin.file:
+        path: "{{ backup_user_info.home }}/.ssh/"
+        state: directory
+        mode: "0700"
+        owner: "{{ borgbackup_user }}"
+        group: "{{ borgbackup_group }}"
+
+    - name: Generate an OpenSSH keypair
```

Comment:
There is also an [issue](https://github.com/borgbase/ansible-role-borgbackup/actions/runs/4456818684/jobs/7827515243?pr=112) with regenerating the key.

  CRITICAL Idempotence test failed because of the following tasks:
  *  => m3nu.ansible_role_borgbackup : Generate an OpenSSH keypair
  *  => m3nu.ansible_role_borgbackup : Set key permission

Reply:
I didn't understand this failure:

the key isn't regenerated if i start the role a second time.

If i disable "Set key permission" the pub key have the wrong permissions: 600 and not 644.

Thanks Con

Reply:
Why is permission 600 wrong for the public key? All my public keys in `.ssh` have 600.